template_feed/Microsoft.DotNet.Common.ItemTemplates/content/BuildTargets/.template.config/ide.host.json

```diff
@@ -0,0 +1,6 @@
+{
+  "$schema": "http://json.schemastore.org/ide.host",
+  "itemHierarchyPaths": ["General"],
+  "appliesTo": ".NET",
+  "icon": "ide/icon.png"
```

Comment:
This should have been automatically discovered, you shouldnt need to specify it in the host file if it is at this path.

Reply:
Good catch - I removed these items, rebuilt, and retested and it all just worked. I think in prior test runs I was either using the wrong version of VS (17.2 stable?) or older versions of the templates (due to the fact I flubbed my uninstall/pack/reinstall loop).